bpf/init.sh

```diff
@@ -108,8 +108,7 @@ function move_local_rules()
 
 function setup_proxy_rules()
 {
-	# Any packet from an ingress proxy uses a separate routing table that routes
-	# the packet back to the cilium host device.
+	# TODO(brb): remove $PROXY_RT_TABLE -related code in v1.14
```

Comment:
Is this comment still relevant? This is going into 1.14, so maybe this should be changed to 1.15?